deps/src/singular.cpp

```diff
@@ -64,13 +64,13 @@ JLCXX_MODULE define_julia_module(jlcxx::Module & Singular)
   Singular.set_const("n_Zp", n_Zp);
   Singular.set_const("n_GF", n_GF);
   Singular.set_const("n_transExt", n_transExt);
-  Singular.set_const("n_Nemo_AnticNumberField", (n_coeffType)17 /*n_Nemo_AnticNumberField*/);
```

Comment:
@hannes14 I wonder why this was done? I am mostly asking because I wonder if there is a fundamental issue or misunderstanding here? This thing should not be necessary at all unless one is for some reason testing with an outdated `Singular_jll` -- then using the enum of course will give a compiler error. But then using the resulting code would result in a crash, so ...?!